### PR TITLE
Adds a preset dronestation event map

### PR DESCRIPTION
**Pull request by Ohagi-Chan**
## About The Pull Request
A quick template for getting dronestation rounds up and running really quickly, just select this map to be used for the next round and reboot the server.
Starts with practically everything the drones will need to get started and even a regenerating supply of basic building materials so admins don't have to spawn them so often.

image
Full map

image
Enlarged

image
Cargo dock and starting area

image
Mining dock

image
Arrivals dock

image
High-yield asteroid

The station offers the basic necessities for a construction drone workforce.
A central hub
Ship docks for a cargo, mining and arrivals shuttle (for when humans arrive)
A full fabrication suite (autolathe, techfab, circuit printer, exosuit fabricator)
Starting materials (100 iron, 100 glass, 75 titanium, a rich asteroid)
A steady income of minerals to avoid reliance on admin spawning (bluespace miner which has a slightly increased ore rate for iron and glass) and a mining shuttle
A research console.
Infinite drone shells.
Basic station constructing tools (RCD, RPD, Floor painter, door painter, decal painter, rapid lighting)

## Why It's Good For The Game
Faster set up for dronestation meme rounds.
Dronestation rounds should be faster and easier for admins to set up and require less maintenance than earlier event rounds.

## Changelog
:cl:
add: new dronestation map
/:cl:

### CODE REVIEW

**Comment by Ohagi-Chan** [comment]
ignore the config changes. they are only to help me test.

File: config/maps.txt [hunk 1]
```diff
@@ -60,10 +60,14 @@ map multiz_debug
...
 
 map austation
-    default
     votable
 endmap
 
+map dronestation
+	default
+	votable
+endmap
```
Comment (TheFakeElon) [inline config/maps.txt:69]: Should not be votable or default.

**Comment by TheFakeElon** [comment]
Is it a template or a map? Those are two very different things

**Comment by Ohagi-Chan** [comment]
> Is it a template or a map? Those are two very different things

it is a map. it is the station map on rounds where it has been chosen.
it should not be permitted as a vote map/ rotation map.

**Comment by Ohagi-Chan** [comment]
> Is it a template or a map? Those are two very different things

I've renamed the PR to reduce confusion.

**Comment by MCterra10** [comment]
it certainly looks cool, but when we ran drone station rounds on austation, there was always a sort of "core" that served as the central hub for hallways and sections to connect. I don't see any of that here. Also the engines and the walls you've chosen to use would contrast really badly with anything the drones chose to build.

**Comment by Ohagi-Chan** [comment]
> it certainly looks cool, but when we ran drone station rounds on austation, there was always a sort of "core" that served as the central hub for hallways and sections to connect. I don't see any of that here. Also the engines and the walls you've chosen to use would contrast really badly with anything the drones chose to build.

Feedback noted; I will prepare another draft that uses more traditional materials for the walls and either swaps the thrusters for ones players are able to make (and disassemble) or I'll forgo them altogether.
I'll have to give the placement of the arrivals/mining area some thought - I agree that it is maybe too close to the starting pod.
The asteroid may have to be reconsidered too.
Cargo will PROBABLY stay attached to the starting pod in the next draft, unless you insist that it should be pushed further away.

**Comment by Ohagi-Chan** [comment]
Converting to a draft until Terra is happy with the layout.

**Comment by MCterra10** [comment]
My main issue is that with the prefab structures around, it's gonna be really hard to start building out the departments. Perhaps if you're interested in having prebuilt infrastructure like docks, you could rip metastation apart and make a skeleton map from it? Drones could be tasked with remaking meta from scratch, or something along those lines

**Comment by Ohagi-Chan** [comment]
I do have a direction left to take this, but I'm feeling unmotivated rn.
Closing until I feel like doing it.

**Pull request closed without merging**